backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/ApplicationRepository.kt

```diff
@@ -109,6 +124,18 @@ object ApplicationRepository {
         }
     }
 
+    fun withdrawApplication(accessKey: String): Boolean {
+        return transaction {
+            val application = ApplicationEntity.find { Applications.accessKey eq accessKey }.single()
+            if (application != null) {
+                application.withdrawalDate = LocalDateTime.now()
```

Comment:
We should probably throw, if it has been withdrawn already.

Reply:
hm but i check onCompleted if transaction return false and send a feedback to the user that it was already withdrawed